diff --git a/pc/Makefile b/pc/Makefile
index 8240afa..2c09c72 100644
--- a/pc/Makefile
+++ b/pc/Makefile
@@ -1,17 +1,17 @@
 all:
 ifeq ($(_WIN32),1)
-	make _WIN32=1 -C iso2opl
-	make _WIN32=1 -C opl2iso
-	make _WIN32=1 -C genvmc
+	$(MAKE) _WIN32=1 -C iso2opl
+	$(MAKE) _WIN32=1 -C opl2iso
+	$(MAKE) _WIN32=1 -C genvmc
 else
-	make -C iso2opl
-	make -C opl2iso
-	make -C genvmc
+	$(MAKE) -C iso2opl
+	$(MAKE) -C opl2iso
+	$(MAKE) -C genvmc
 endif
 
 clean:
-	make -C iso2opl clean
-	make -C opl2iso clean
-	make -C genvmc clean
+	$(MAKE) -C iso2opl clean
+	$(MAKE) -C opl2iso clean
+	$(MAKE) -C genvmc clean
 
 rebuild: clean all
